### PR TITLE
Submission -  FAB menu animation - Harin

### PR DESCRIPTION
**Pull request by harintrivedimm**
#### Demonstration Video
| FAB Menu Animation  | 
| ------------- | 
| <video src='https://github.com/mutualmobile/flutter_animations_example/assets/64526025/1a67b832-d32e-44a3-bd60-56a2034dc142' width=180/> |

#### Details
* This PR adds new screen which demonstrates animation demo for  custom FAB menu items and circular reveal animation

#### Steps to Test
* Select direction and tap on FAB to toggle menu. Click on any item to reveal the view with selected option

### CODE REVIEW

**Comment by harintrivedimm** [comment]
Demonstration Video:
https://github.com/mutualmobile/flutter_animations_example/assets/64526025/015bac37-858a-449b-a391-5e60c887b1a3

**Comment by YugeshJainMM** [comment]
https://github.com/mutualmobile/flutter_animations_example/assets/98592058/37c15e55-353e-4f3b-9275-c065f7b4faa9

**Comment by pushpalroymm** [comment]
@harintrivedimm Why is the PR showing only 2 file changes? The animation widget file is not included in the PR?

cc: @shubhamsinghmutualmobile @aditya-bhawsar-mm

**Comment by harintrivedimm** [comment]
> @harintrivedimm Why is the PR showing only 2 file changes? The animation widget file is not included in the PR?

PR updated @pushpalroymm

File: pubspec.lock
Comment (aditya-bhawsar-mm) [inline pubspec.lock:1]: no need to push this file

**Review by aditya-bhawsar-mm — APPROVED** [review]

**Pull request merged**